File: Solution/CoMentor/UserStoryImpl/Java/US_SME_55/Backend/src/main/java/com/transflower/tflcomentor/Services/RuntimesService.java
package com.transflower.tflcomentor.Services;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import com.transflower.tflcomentor.Dtos.RuntimesDTO;
import com.transflower.tflcomentor.Repositories.IRuntimesRepository;

@Service
public class RuntimesService implements IRuntimesService {

    @Autowired
    private IRuntimesRepository runtimesRepository;

    @Override
    public boolean addRuntime(RuntimesDTO runtimedto) {
        return runtimesRepository.addRuntime(runtimedto);
    }

}

File: Solution/CoMentor/Integrated/dotnetintegrated/backend/Services/Implementations/RuntimesService.cs

using backend.DTOs;

using backend.Repositories.Interfaces;

namespace backend.Services
{
    public class RuntimesService : Interfaces.IRuntimesService
    {
        private readonly IRuntimesRepository _repo;

        public RuntimesService(IRuntimesRepository repo)
        {
            _repo = repo;
        }

        public Task<List<RuntimeDto>> GetRuntimes()
            => _repo.GetRuntimes();

       
    }
}